earthpy/tests/test_plot.py

```diff
@@ -0,0 +1,86 @@
+"""Tests for the plot module"""
+
+import numpy as np
+import pytest
+import earthpy.spatial as es
+import matplotlib as mpl
+
+
+
+""" General functions for matplotlib elements """
+
+
+# Create tuple
+tuups = (1, 2)
+
+im = np.random.randint(10, size=(2, 4, 5))
+single_band = im[0]
+
+def test_arr_parameter():
+    """Error raised is not provided with an array"""
```

Comment:
The docstring for this function took me a few reads to understand. How about something like "If an array is not provided, raise an AttributeError"